NEWS.md

```diff
@@ -0,0 +1,18 @@
+# synapser 0.3
+
+## New features
+
+* New convenience function `synBuildTable` creates a Table Schema based on the given data, and returns a Table object that can be stored in Synapse using `synStore`.
+* New convenience function `synMove` allows users to move entities to a different parent.
+
+## Minor bug fixes and improvements
+
+* `synStore` now uses a single thread to avoid the C stack limit error (SYNR-1323).
```

Comment:
I would replace the Jira IDs with links to the Jira issues.   If you do this, make sure the issues are viewable by anonymous Jira users.